### ISSUE TITLE
ktest: default results = programs

### ISSUE THREAD

**Issue by grosu** [kframework/k-legacy#22]
There are two options in ktest which always require folders: -programs and -results.  By default, both of these are set to the current directory ".".  Many users  will likely hold the results in the same folder with the programs, either as separate subfolders or even just flat in the same directory.

To save the user from writing the same folder twice in most cases, e.g.

ktest ... -programs FOLDER -results FOLDER

it would be nice to have the -results folder be by default equal to the -programs folder, so one would only write

ktest ... -programs FOLDER

**Comment by andreiarusoaie** [comment]
Done.

**Comment by andreiarusoaie** [comment]
I think I fixed it now. If you omit the results tag in the config.xml then the results folder is the same as the programs folder. I've seen that in simple-untyped Grigore already had some in/out files so I gave it a try:

fmse:k$ pwd
/home/andrei.arusoaie/work/k
fmse:k$ cat tests/config.xml 
<tests>



  <test
      definition="dist/tutorial/2_languages/1_simple/1_untyped/simple-untyped.k"
      programs="dist/tutorial/2_languages/1_simple/1_untyped/programs"
      extension="simple"
      exclude="diverse/dekker.simple"
      />
</tests>

fmse:k$ ktest tests/config.xml
Buildfile: tests/config.xml
Kompiling the language definitions...(1 in total)
SUCCESS
Generating PDF documentation...(1 in total)
SUCCESS
Running /home/andrei.arusoaie/work/k/dist/tutorial/2_languages/1_simple/1_untyped/simple-untyped.k programs... (33 in total, 6 with input, 23 with output)
SUCCESS
fmse:k$ 

On the other hand, it also works if you run it in the simple-untyped folder:

fmse:1_untyped$ pwd
/home/andrei.arusoaie/work/k/dist/tutorial/2_languages/1_simple/1_untyped
fmse:1_untyped$ ktest simple-untyped.k -programs programs -extension simple -exclude diverse/dekker.simple 
Kompiling the language definitions...(1 in total)
SUCCESS
Generating PDF documentation...(1 in total)
SUCCESS
Running simple-untyped.k programs... (33 in total, 6 with input, 23 with output)
SUCCESS
fmse:1_untyped$ 

I hope this is the right behavior.
Note that I changed 'extensions' into 'extension' and I updated the config.xml file.
